ipatests/test_integration/test_external_ca.py

```diff
@@ -218,3 +217,62 @@ def test_external_ca_dirsrv_stop(self):
         tasks.kinit_admin(self.master)
         result = self.master.run_command(['ipa', 'user-show', 'admin'])
         assert 'User login: admin' in result.stdout_text
+
+
+class TestExternalCAInvalidCert(IntegrationTest):
+    """Manual renew external CA cert with invalid file"""
+
+    def test_external_ca(self):
+        # Step 1 of ipa-server-install.
+        result = install_server_external_ca_step1(self.master)
```

Comment:
Don't assign to ``result`` unless you need the object later. You can and should remove ``result = `` in five out of six occasions. IIRC the code linter won't accept unused assignments, too.